src/ts/components/seekbar.ts

```diff
@@ -577,7 +615,10 @@ export class SeekBar extends Component<SeekBarConfig> {
       this.setSeekPosition(targetPercentage);
       this.setPlaybackPosition(targetPercentage);
       this.onSeekPreviewEvent(targetPercentage, true);
+
+      setAriaSliderValue(e);
```

Comment:
Is it intended that the value changes when the user just hovers the bar?

Why exactly does this value mean? The current position? Or also preview values?

Reply:
Defines only the current position so users with screen readers would know where they are located on the slider

Reply:
What I was thinking about is assume we have our default seekbar during a VOD (0 - 100). The `currentTime` is about 10 but the user now hovers over the seekbar at position 50. What should this value be now? 10 where we are currently playing, or 50 where the user hovers over 🤔

Reply:
Only if the slider is pressed and then moved, the current value will be displayed

Reply:
👍